### ISSUE TITLE
Handling  the multiple Docker images and organization.

### ISSUE THREAD

**Issue by envygeeks** [jekyll/jekyll#3702]
Because  I want us to have multiple docker images in multiple styles (AKA a server[production], development[master,production,beta], production[stable]) I want to put it to the community how we handle this situation.

Right now we have docker-jekyll (recently renamed from docker) but I think it would be better to have docker-jekyll (stable) docker-jekyll-master (master - built every time Git is updated.), docker-jekyll-beta (maybe just drop this one and use master because it's the same thing but updated more often)

In other words I think we should repo each type and make them their own images, while this means I can no-longer cherry-pick changes into the branches it does mean that things become easier to track down when it comes to the image you are using.

Comments?
/cc @jekyll/core

**Comment by parkr** [comment]
Can we put everything in one repo and have subdirectories for each variation? Is `git clone` part of the deployment/build process? Might make it easier for Hulk, too.

**Comment by envygeeks** [comment]
Hmmm... that might work because then I can ln -s anything I want to share from a shared/ folder and override anything I don't want to share in the copy/ folder of the image.  That's a good idea, I might try that this afternoon after I finish the Octopress image.
